app/assets/javascripts/shared/inline_editing.js

```diff
@@ -63,21 +74,26 @@ function initInlineEditing(title) {
       return true;
     }
 
-    $editBlock.click(e => {
+    $editBlock.click((e) => {
+      if (e.target.tagName === 'A') return true;
```

Comment:
Please comment this, or it's hocus pocus. Which element is 'A'?

Reply:
'A' it is tag name of <a></a> element